src/context/mod.rs

```diff
@@ -22,6 +22,12 @@ pub trait Context {
     /// Calls the function that is linked to the given identifier with the given argument.
     /// If no function with the given identifier is found, this method returns `EvalexprError::FunctionIdentifierNotFound`.
     fn call_function(&self, identifier: &str, argument: &Value) -> EvalexprResult<Value>;
+
+    /// Checks if builtin function has been disabled.
+    fn is_builtin_fn_disabled(&self) -> bool;
+
+    /// Disables Builtin function.
+    fn disable_builtin_fn(&mut self);
```

Comment:
I made it so that EmptyContext can't have builtin functions enabled while EmptyContextWithBuiltinFunctions can't have builtin functions disabled. 
They return EvalexprError::InvalidContext error when calling the setter function.